### ISSUE TITLE
Health bar is still there. Picture included

### ISSUE THREAD

**Issue by Gamerbolts** [Flenix/HUDWidgets#8]
http://imgur.com/a/GMg9U

Everything works perfectly except the heath is in the middle which is problematic.

**Comment by Fureniku** [comment]
This looks like it's being caused by another mod that overrides the health bar for some reason - possibly an outdated Tinkers or maybe another mod (It's common for any mod which increases/modifies Health).
I know Tinkers now supports HUDwidgets, but not sure which version it was added in, but it could be another mod too - can you list the mods you have installed?

At any rate it's something they will have to fix, I hide the vanilla HUD elements on a case by case basis, but if another mod re-adds it for a custom implementation I can't do anything :(

**Comment by Gamerbolts** [comment]
Here is my list of mods

https://docs.google.com/spreadsheets/d/16V8wYCzGRSHEAsuIk1tNo7cQBZSlljk-K2dOtc2_slA/edit?usp=sharing

Trying to think what other mod could be altering my health.

**Comment by Fureniku** [comment]
That's a hell of a list...
Just for the sake of testing, try removing Tinkers (and things that depend on it), and see if that does resolve it. If so I can give you my Tinkers build which I know 100% works.If that doesn't fix it, it'll basically be looking through the mods for anything which for example adds more health via various ways, there's a lot of mods I'm not that familiar with on there but for example does Thaumcraft have a spell to increase health or something along those lines?There's also an armour HUD thing you have, which may tweak health for certain configurations (Just like in HUDWidgets you can display armour on the health bar, they may do something similar)

Date: Fri, 2 Sep 2016 16:37:13 -0700

**Comment by Gamerbolts** [comment]
Yep over 300 mods, been working on it for about 3 months now. 

Ok I will start with Tinkers and go on from there and see if I can pinpoint the mod

**Comment by Fureniku** [comment]
Alright. When I have to trial and error something like this I tend to remove mods in sets of 10 just to save time narrowing it down - with 300 you might want to try 50 or so then narrow down from there though ;)

Date: Fri, 2 Sep 2016 16:45:38 -0700

**Comment by Gamerbolts** [comment]
Thankfully it won't come to that. I removed Tinkers Construct and Tinkers Mechworks. And the health bar is fixed. I have the newest version of Tinkers I believe.

But yeah that is how I have been testing conflicts take 50 mods at a time. Painfully slow but that's what I get for creating a massive modpack.

**Comment by Gamerbolts** [comment]
My version of Tconstruct is 1.8.8. You said you have a version that will definitely work? Cause it works without TConstruct but I would really like to have that mod.

**Comment by Fureniku** [comment]
Ah yeah, just looked back and they didn't merge my changes because they had "no plans to release any more 1.7.10 versions"
My tweaked build is 1.8.5, there's not that much in the way of changes between them (Almost entirely bug fixes for bugs I've not encountered in the year+ I've had this build)

[TConstruct-1.7.10-1.8.5.zip](https://github.com/Flenix/HUDWidgets/files/453619/TConstruct-1.7.10-1.8.5.zip)

**Comment by Gamerbolts** [comment]
So much for their no plans to release anymore 1.7.10 versions lol. Awesome I will download that and put it in :) Thank you so much. As for the other issue with the vertical bar, got a backlash of people who didn't want the bar vertical so reverted it back to horizontal so no rush on that :)

Thanks again!

**Comment by Fureniku** [comment]
Alright; let me know if it does/doesn't work, as I may of done some other witchery that I forgot about (Built this pack about 18 months ago)
Yeah, vertical hotbars are incredibly unpopular, which I guess is why you're the first to notice that. I'll still fix it, as once people adapt it would make the GUI better, but I dunno if anyone will ever use it ;)

Date: Sat, 3 Sep 2016 14:08:26 -0700

**Comment by Gamerbolts** [comment]
Will let you know when I get to testing. Yeah had no idea how unpopular vertical huds were lol. But good idea to update just in case :)

**Comment by Gamerbolts** [comment]
The older version works. But unfortunately when I load the world and go to press W to move the Inventory screen pops up. when I close the window it just pops up again, I removed TConstruct and it worked again, I researched and found that the older versions of tinkers had this issue

**Comment by Fureniku** [comment]
Hmm, that's very weird. Are you using that key modifier mod, Not Enough Keys I think it was? One of the bugfixes mentioned in the changelog since my version was for that.
Unfortunately, they seem to have moved the code which renders the health bar in their source, and Github won't let you search branches other than master (A branch is a sort of variation of code; most developers make a new branch for each MC version so that if they need to they can return and fix bugs in the old versions) - so I can't find it anywhere to make a new fixed version =/

Date: Sat, 3 Sep 2016 16:53:07 -0700

**Comment by Gamerbolts** [comment]
yep I am using Not Enough Keys. That sucks that you won't be able to fix it, but I guess not enough keys is not that big of a loss. i'll test it

**Comment by Gamerbolts** [comment]
Shoot that version of Tinkers didn't fix it :( the hearts are still at the bottom. but removing notenoughkeys did work for the annoying menu thing

**Comment by Gamerbolts** [comment]
The hearts are still there with that version of TConstruct. It's not a real big issue now that i moved my hotbar horizontal. I just have two places with health indicators lol.
